cns/service/main.go

```diff
@@ -1013,6 +1016,25 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 		return errors.Wrapf(err, "failed to setup reconciler with manager")
 	}
 
+	mux := httpRestServiceImplementation.Listener.GetMux()
```

Comment:
we have to remember to update the metrics annotation port when we bring this in. Weee non atomic changes.